verifier/src/evidence.rs

```diff
@@ -317,23 +324,33 @@ impl<'a, C: CertificateChainVerifier, E: Accessor<Evidence<Vec<u8>>>> Verifier<E
         let quote = &evidence.quote;
 
         let (tcb_key, tcb_chain_verification) = self.verify_tcb_signing_chain(collateral);
-        let (_, qe_chain_verification) = self.verify_qe_identity_signing_chain(collateral);
+        let (qe_key, qe_chain_verification) = self.verify_qe_identity_signing_chain(collateral);
         let (_, quote_chain_verification) = self.verify_quote_signing_chain(quote, collateral);
 
         let tcb_info_verifier = SignedTcbInfoVerifier::new(tcb_key, self.time);
         let tcb_info_verification = tcb_info_verifier.verify(&evidence);
 
+        let qe_identity_verifier = SignedQeIdentityVerifier::new(qe_key, self.time);
+        let qe_identity_verification = qe_identity_verifier.verify(&evidence);
+
+        let qe_report_body_verifier = QeReportBodyVerifier::new(evidence.qe_identity.clone());
+        let qe_report_body_verification = qe_report_body_verifier.verify(&evidence);
```

Comment:
The verifier could have been a member in `EvidenceVerifier`, but I kept it here for consistency with the other verifiers being created here.